app/sequencescape/sequencescape/api/v2.rb

```diff
@@ -7,6 +7,7 @@ module Sequencescape::Api::V2
   # transfer_request_as_target.source_asset is added to the includes to make
   # the source receptacles available in presenters.
   PLATE_PRESENTER_INCLUDES = [
+    :ancestors,
```

Comment:
@andrewsparkes and @StephenHulme I added this to include the ancestors when we get a plate for a presenter.  This didn't fix it on its own though, so see the other comment I'm about to make.